tests/system/action/user/test_create_saml_account.py

```diff
@@ -0,0 +1,84 @@
+from tests.system.action.base import BaseActionTestCase
+
+
+class UserCreateSamlAccount(BaseActionTestCase):
+    def test_create_saml_account_correct(self) -> None:
```

Comment:
This test is not needed, if we have the second one

Reply:
Removed then.